nano.el

```diff
@@ -15,12 +15,6 @@
 ;; You should have received a copy of the GNU General Public License
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 ;; ---------------------------------------------------------------------
-(package-initialize)
```

Comment:
Why did you remove this line?

Reply:
Oh, I thought that it was also unneeded. I can put it back in.